src/GitUtils.ps1

```diff
@@ -456,11 +456,17 @@ function Start-NativeSshAgent([switch]$Quiet, [string]$StartupType = 'Manual') {
         Start-Service "ssh-agent"
     }
 
-    # Make sure GIT_SSH is set to OpenSSH-Win32
-    setenv "GIT_SSH" (Get-Command ssh.exe -ErrorAction Ignore | Select-Object -ExpandProperty Path)
-    
-    if (!$Quiet) {
-        Write-Host "Setting GIT_SSH set to $($env:GIT_SSH)."
+    # Make sure git is configured to use OpenSSH-Win32
+    $hasSshExplicitlySet = git config --global core.sshCommand
+
+    if (!$hasSshExplicitlySet) {
```

Comment:
Should we warn folks if their existing `core.sshCommand` looks like it's wrong relative to the `ssh.exe` we found?

Reply:
Good idea, I've modified this to output a warning if it's set to something other than what we expect.